add itemRunner method getData

lecosson opened the pull request:
Related to: https://oat-sa.atlassian.net/browse/TAO-9637
Acces the data about APIP the content and related media files

#### Steps to reproduce
prepare TAO instance, go to taoItems/views/package.json and replace current depencies
`  "dependencies": {
    "@oat-sa/tao-item-runner": "0.3.1"
  }`
with this branch:
`  "dependencies": {
    "@oat-sa/tao-item-runner": "github:oat-sa/tao-item-runner-fe#feauture/TAO-9637-access-apip-data"
  }`

login to prepared delivery and run test with any enabled tool plugin (for example, highlighter), set breakpoint inside any method executed after test loading and check data availability in console:
> testRunner.itemRunner.getData().apipAccessibility

or if testRunner variable unavailable:
>this.getTestRunner().itemRunner.getData().apipAccessibility

method must return null or object with Apip data if it presented in test data

#### Dependencies
Requires : nothing

lecosson committed:
add itemRunner method getData
https://oat-sa.atlassian.net/browse/TAO-9637 ([ACT] [TTS - FE] Acces the data about APIP the content and related media files)

krampstudio commented:
Maybe you can add the method to the doc in https://github.com/oat-sa/tao-item-runner-fe/blob/master/src/runner/README.md  and then I can validate and release it.

lecosson committed:
add itemRunner method getData - add method description to readme.MD
https://oat-sa.atlassian.net/browse/TAO-9637 ([ACT] [TTS - FE] Acces the data about APIP the content and related media files)
